components/ItemCard/index.js

```diff
@@ -5,28 +5,137 @@ import {
+      });
+      if (response.ok) {
+        closeAllPopups();
+        mutate();
+      } else {
+        alert("Fehler beim setzen des neuen Wertes");
+      }
+    } catch (error) {
+      alert("Fehler beim Zugriff auf Datenbank");
+    }
+  }
+
   return (
-    <PartsListContainer borderColor="var(--color-item)">
-      <PartCardFlexContainer direction="row" justify="flex-start">
-        <PartCardImage
-          src={item.imgUrl}
-          alt={item.name}
-          width={100}
-          height={100}
-        />
-        <PartCardFlexContainer direction="column" justify="flex-start">
-          <PartCardText>{item.dateAssembled}</PartCardText>
-          <PartCardText>
-            {item.totalPurchasingPrice} {item.currency}
-          </PartCardText>
+    <>
+      <PartsListContainer borderColor="var(--color-item)">
+        <PartCardFlexContainer direction="row" justify="space-between">
+          <PartCardImage
+            src={item.imgUrl}
+            alt={item.name}
+            width={100}
+            height={100}
+          />
+          <PartCardFlexContainer direction="column" justify="flex-start">
+            <PartCardText>{item.dateAssembled}</PartCardText>
+            <PartCardText>
+              {item.totalPurchasingPrice} {item.currency}
+            </PartCardText>
+            {/* Ternarys for correct display of Price-information depending on which data has been given */}
+            {item.targetPrice && !item.soldForPrice ? (
+              <>
+                <PartCardText>
+                  {"VK(soll): "}
+                  {item.targetPrice} {item.currency}
+                </PartCardText>
+                <PartCardText>
+                  {"Gewinn(soll): "}
+                  {item.targetPrice - item.totalPurchasingPrice} {item.currency}
+                </PartCardText>
+              </>
+            ) : null}
+            {item.targetPrice && item.soldForPrice ? (
+              <>
+                <PartCardText>
+                  {"VK(soll): "}
+                  {item.targetPrice} {item.currency}
+                </PartCardText>
+                <PartCardText>
+                  {"VK(ist): "}
+                  {item.soldForPrice} {item.currency}
+                </PartCardText>
+                <PartCardText>
+                  {"Gewinn(ist): "}
+                  {item.soldForPrice - item.totalPurchasingPrice}{" "}
+                  {item.currency}
+                </PartCardText>
+              </>
+            ) : null}
+            {!item.targetPrice && item.soldForPrice ? (
+              <>
+                <PartCardText>
+                  {"VK(ist): "}
+                  {item.soldForPrice} {item.currency}
+                </PartCardText>
+                <PartCardText>
+                  {"Gewinn(ist): "}
+                  {item.soldForPrice - item.totalPurchasingPrice}{" "}
+                  {item.currency}
+                </PartCardText>
+              </>
+            ) : null}
```

Comment:
for any later refactirong round, you could check if there is a possibility to make components / a component for all the rendering options here